portfolio/src/main/webapp/image-upload.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+            <title>Andrew Pham | Images</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+    </head>
+    <body onload="fetchBlobstoreUrlAndDisplayMemePosts()">
+        <form id="my-form" enctype="multipart/form-data" method="POST">
```

Comment:
"my-form" doesnt really give any information about what this is for. Something like "image-upload" is more informative (thats what this is right?)

Reply:
Yeah, I changed it to meme-form to be more informative.